### PR TITLE
Add the multilingual Exams dataset

### PR DESCRIPTION
**Pull request by yjernite**
https://github.com/mhardalov/exams-qa

`multilingual` configs have all languages mixed together

`crosslingual` mixes the languages for test but separates them for train and dec, so I've made one config per language for train/dev data and one config with the joint test set

### CODE REVIEW

**Comment by yjernite** [comment]
Will slim down the dummy files in the morning

File: datasets/exams/exams.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+        super(ExamsConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+        self.lang = lang
+        self.with_para = "_with_para" if with_para else ""
+
+
+class Exams(datasets.GeneratorBasedBuilder):
+    """Exams dataset"""
+
+    VERSION = datasets.Version("1.0.0")
+
```
Comment (lhoestq) [inline datasets/exams/exams.py:109]:
```suggestion
    BUILDER_CONFIG_CLASS = ExamsConfig
```

File: datasets/exams/exams.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+                    },
+                    "answerKey": datasets.Value("string"),
+                    "info": {
+                        "grade": datasets.Value("int32"),
+                        "subject": datasets.Value("string"),
+                        "language": datasets.Value("string"),
+                    },
+                }
+            )
+        print(features)
```
Comment (lhoestq) [inline datasets/exams/exams.py:189]:
```suggestion
```

**Review by lhoestq — APPROVED** [review]
really cool !

Feel free to merge after making the dummy files lighter and if the CI passes

**Pull request merged**